rest/data/project.go

```diff
@@ -267,7 +267,17 @@ func (pc *DBProjectConnector) CopyProjectVars(oldProjectId, newProjectId string)
 	return errors.Wrapf(vars.Insert(), "error inserting variables for project '%s", newProjectId)
 }
 
-func (ac *DBProjectConnector) GetProjectEventLog(id string, before time.Time, n int) ([]restModel.APIProjectEvent, error) {
+func (ac *DBProjectConnector) GetProjectEventLog(project string, before time.Time, n int) ([]restModel.APIProjectEvent, error) {
+	id, err := model.GetIdForProject(project)
+	if err != nil {
+		grip.Debug(message.WrapError(err, message.Fields{
```

Comment:
in my mind returning an error also preserves existing behavior, since if an _id is passed in it would just get returned the same right?

Reply:
If someone passed in a project Id for a project that didn't exist, we would just return empty events, not an error. We don't validate anywhere else if the project actually exists.